packages/web-components/src/radio-group/radio-group.ts

```diff
@@ -0,0 +1,64 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTRadioGroup } from '@microsoft/fast-foundation';
+
+/**
+ * The base class used for constructing a fluent-radio-group custom element
+ * @public
+ */
+export class RadioGroup extends FASTRadioGroup {
+  /**
+   * Describes the ID of the element that labels the current element, as used for accessibility with screen readers.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: aria-labelledby
+   */
+  @attr({ attribute: 'aria-labelledby' })
+  public ariaLabelledby: string = '';
+
+  /**
+   * sets radio layout styles
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: stacked
+   */
+  @attr({ attribute: 'stacked', mode: 'boolean' })
+  public stacked: boolean = false;
+
+  protected layoutChanged(): void {
+    if (!this.$fastController.isConnected) {
+      return;
+    }
+    if (this.slottedRadioButtons !== undefined) {
+      this.slottedRadioButtons.forEach((item: HTMLElement, index: number) => {
+        if (this.stacked && this.orientation === 'horizontal') {
+          item.setAttribute('stack', '');
+        } else {
+          item.removeAttribute('stack');
+        }
+      });
+    }
+  }
+
+  protected disableChanged(): void {
+    if (!this.$fastController.isConnected) {
+      return;
+    }
+    this.slottedRadioButtons.forEach((item: HTMLElement, index: number) => {
+      if (this.disabled) {
+        item.setAttribute('disabled', '');
+      }
+    });
+  }
```

Comment:
I'd like to understand the addition of this here - for awareness, the iteration over individual radio buttons and modifying their state was explicitly removed with a purpose in this PR: https://github.com/microsoft/fast/pull/6601

The corresponding issue is here: https://github.com/microsoft/fast/issues/6567

(If this hasn't been published we plan on doing so by EOW)

Reply:
@chrisdholt I did this because slotted content within Radio needs specific disabled styling. But is it possible to target slotted content within slotted content from the RadioGroup?

Something like this..?

```
// markup

<fluent-radio-group disabled>
      slotted:
           <fluent-radio>
                slotted: 
                     <div class="label"> ...</div>
            </fluent-radio>
</fluent-radio-group>

// radiogroup.styles.ts

:host([disabled]) ::slotted([role="radio"]) ::slotted([.label]) {
   // apply styles
}
```

Reply:
What is the disabled styling you need to accomplish?

Reply:
I'll pull this down and check it out.

Reply:
@chrisdholt This might be a good case for local tokens? It would be much more straight forward then the Menu + Menu Item attempt.

Reply:
@procload i want to reiterate that this will override any state set outside the component.